pixeltable/io/label_studio.py

```diff
@@ -18,6 +18,15 @@
 from pixeltable.io.external_store import Project, SyncStatus
 from pixeltable.utils import coco
 
+# label_studio_sdk>=1 and label_studio_sdk<1 are not compatible, so we need to try
+# the import two different ways to insure intercompatibility
+try:
+    # label_studio_sdk<1 compatibility
+    import label_studio_sdk.project as ls_project  # type: ignore
+except ImportError:
+    # label_studio_sdk>=1 compatibility
+    import label_studio_sdk._legacy.project as ls_project  # type: ignore
```

Comment:
Given that this is marked 'legacy' in the newer version, is there a newer interface we should use?